backend/isic/isic_crawler.py

```diff
@@ -76,21 +75,23 @@ def ISIC_getdata(numjson=20, numimage=20,
         image_data = {}
         datalist = []
         i = 0
+        dir = path+"/static/isic/"
+        if not os.path.exists(dir):
+            os.makedirs(dir)
         for set in dataset:
             if(dermo_filter is False or check_dermoscopic(set) and
                     melano_filter is False or check_melanoma(set)):
                 image_data['Name'] = str(set['name'])
                 image_data['Caption'] = str(set['_id'])
-                headers = {
-                    'Accept': 'application/json'
-                }
-                r = requests.get("https://isic-archive.com/api/v1/image/" +
-                                 str(set['_id'])+"/download", headers=headers)
-                path = str(pathlib.Path(__file__).parent.absolute())
-                if not os.path.exists(path+"/Image"):
-                    os.makedirs(path+"/Image")
-                img = Image.open(BytesIO(r.content))
-                img.save(path+"/Image/"+str(image_data['Name'])+'.jpg')
+                if not os.path.exists(dir+image_data['Name']+'.jpg'):
+                    headers = {
+                        'Accept': 'application/json'
+                    }
+                    r = requests.get("https://isic-archive.com/api/v1/image/" +
+                                     str(set['_id'])+"/download",
+                                     headers=headers)
+                    img = Image.open(BytesIO(r.content))
+                    img.save(dir+image_data['Name']+'.jpg')
```

Comment:
Nitpick: using f-string to make it look nicer.

```suggestion
                    img.save(f'{dir}{image_data['Name']}.jpg')
```